lib/fastlane/plugin/wpmreleasetoolkit/helper/promo_screenshots_helper.rb

```diff
@@ -233,7 +235,7 @@ def draw_text_to_canvas(canvas, text, width, height, x_position, y_position, fon
         begin
           tempTextFile = Tempfile.new()
 
-          command = "bundle exec drawText html=\"#{text}\" maxWidth=#{width} maxHeight=#{height} output=#{tempTextFile.path} fontSize=#{font_size} stylesheet=\"#{stylesheet_path}\" alignment=\"#{position}\""
+          command = "drawText html=\"#{text}\" maxWidth=#{width} maxHeight=#{height} output=#{tempTextFile.path} fontSize=#{font_size} stylesheet=\"#{stylesheet_path}\" alignment=\"#{position}\""
 
           UI.crash!('Unable to draw text') unless system(command)
```

Comment:
Also could be worth converting this to `sh('drawText', 'html="#{text}"', 'maxWidth=#{width}', …)` at some point, as it would prevent both parsing and security issues (via injection) and also be more idiomatic and also allow fastlane's build-in handling of output and error for shell commands to take over.

This can probably be for a separate PR though (but worth keeping in mind).

Reply:
Good call – addressed in 36f4d6a04fdb7b2cad7928da4823a6487051a9bb.